DOL.WHD.Section14c.Web/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": ["es2015", "dom"],
+    "noImplicitAny": true,
```

Comment:
I'm getting a number of red squigglies in Atom related to variables having an implicit `Any` type.  Is that related to this rule?  And if so, should those be addressed?

Reply:
hmmm, i'm not sure (don't have atom) -- let me change that rule and let me know if the red goes away